packages/suite/src/views/wallet/tokens/common/TokenList.tsx

```diff
@@ -0,0 +1,285 @@
+                            )}
+                            <CryptoAmount
+                                value={t.balance}
+                                customSymbol={
+                                    network.networkType === 'cardano'
+                                        ? undefined
+                                        : blurUrls(t.symbol?.toUpperCase())
+                                }
+                            />
+                        </Amount>
+                    </Cell>
+                    {!hideRates && (
+                        <>
+                            <Cell>
+                                <PriceTickerWrapper>
+                                    <PriceTicker
+                                        symbol={network.symbol}
+                                        contractAddress={t.contract as TokenAddress}
+                                    />
+                                </PriceTickerWrapper>
+                            </Cell>
+                            <Cell>
+                                <TrendTicker
+                                    symbol={network.symbol}
+                                    contractAddress={t.contract as TokenAddress}
+                                />
+                            </Cell>
+                        </>
+                    )}
+                    <Cell $isActions>
+                        <Dropdown
+                            alignMenu="bottom-right"
+                            items={[
+                                {
+                                    key: 'export',
+                                    options: [
+                                        {
+                                            label: <Translation id="TR_NAV_SEND" />,
+                                            onClick: () => {
+                                                goToWithAnalytics('wallet-send', {
+                                                    preserveParams: true,
+                                                });
+                                            },
+                                            isHidden: !isMobileLayout,
+                                        },
+                                        {
+                                            label: <Translation id="TR_NAV_RECEIVE" />,
+                                            onClick: () => {
+                                                goToWithAnalytics('wallet-receive', {
+                                                    preserveParams: true,
+                                                });
+                                            },
+                                            isHidden: !isMobileLayout,
+                                        },
+                                        {
+                                            label: <Translation id="TR_HIDE_TOKEN" />,
+                                            onClick: () => console.log(''),
```

Comment:
It seems that the logic for hiding tokens is completely missing here. Did you forget to push the latest changes, or is this key functionality still not implemented?